scrapper.py

```diff
@@ -44,20 +77,111 @@ def get_search_urls(self):
         pass
 
 
+class HTMLParser:
+    def __init__(self, article_url, article_id):
+        self.article_url = article_url
+        self.article_id = article_id
+        self.article = Article(self.article_url, self.article_id)
+
+    def _fill_article_with_meta_information(self, article_bs):
+        # TITLE
+        self.article.title = article_bs.find('div', class_='entry__title').find('h1').text.strip()
+        # AUTHOR
+        try:
+            self.article.author = article_bs.find('span', class_='entry__author').text.strip()
+        except AttributeError:
+            self.article.author = "NOT FOUND"
+        # DATE
+        raw_date = article_bs.find('div', class_='time').text
+        months_dict = {'января': '01',
+                       'февраля': '02',
+                       'марта': '03',
+                       'апреля': '04',
+                       'мая': '05',
+                       'июня': '06',
+                       'июля': '07',
+                       'августа': '08',
+                       'сентября': '09',
+                       'октября': '10',
+                       'ноября': '11',
+                       'декабря': '12',
+                       }
+        for month in months_dict:
+            if month in raw_date:
+                raw_date = raw_date.replace(month, months_dict[month])
+            elif month.capitalize() in raw_date:
+                raw_date = raw_date.replace(month.capitalize(), months_dict[month])
+        self.article.date = datetime.strptime(raw_date, '%d %m %Y %H:%M')
+        # TOPICS
+        topics_bs = article_bs.find('div', class_='entry__tags').find_all('a')
+        for topic_bs in topics_bs:
+            self.article.topics.append(topic_bs.text[1:])
+
+    def _fill_article_with_text(self, article_bs):
+        text_of_article = ''
+        texts = article_bs.find('article', class_='entry__body').find_all('p')
+        for text_part in texts:
+            text_of_article += text_part.text
+        self.article.text = text_of_article
+
+    def parse(self):
+        response = requests.get(url=self.article_url)
+        article_bs = BeautifulSoup(response.text, 'lxml')
+
+        self._fill_article_with_text(article_bs)
+        self._fill_article_with_meta_information(article_bs)
+        return self.article
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    path = Path(base_path)
+    if path.exists():
+        shutil.rmtree(path)
+    path.mkdir(parents=True)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path) as file:
+        config = json.load(file)
+
+    for url in config["seed_urls"]:
```

Comment:
what if `config["seed_urls"]` is not iterable? e.g. an integer

Reply:
it now handles this